var/spack/repos/builtin/packages/libdrm/package.py

```diff
@@ -33,6 +33,6 @@ def configure_args(self):
             # Needed to fix build for spack/spack#1740, but breaks newer
             # builds/compilers
             args.append('LIBS=-lrt')
-        if self.spec.satisfies('%gcc@10.0.0:'):
+        if self.spec.satisfies('%gcc@10.0.0:') or self.spec.satisfies('%clang@12.0.0'):
```

Comment:
```suggestion
        if self.spec.satisfies('%gcc@10.0.0:') or self.spec.satisfies('%clang@12.0.0:'):
```

Reply:
A prominent case of self-suggestion on Github :monocle_face:

Reply:
What can I say, I can be very persuasive!